internal/executor/instance/persistentworker/isolation/tart/tart.go

```diff
@@ -88,53 +93,26 @@ func (tart *Tart) Run(ctx context.Context, config *runconfig.RunConfig) (err err
 		})
 	}
 
-	tmpVMName := fmt.Sprintf("%s%d-", vmNamePrefix, config.TaskID) + uuid.NewString()
+	tmpVMName := fmt.Sprintf("%s-warm-", vmNamePrefix) + uuid.NewString()
```

Comment:
This seems to add the `-warm-` string to *all* Tart instances and not just the warm ones.

Should we add an additional argument to `Warmup()` that will specify this optional string to be added to VM names in https://github.com/cirruslabs/cirrus-cli/pull/700?

Reply:
Good idea. Maybe allow to optinally specify `vmNamePrefix` as `"cirrus-warm"` vs the default `"cirrus"`?

Reply:
Added in https://github.com/cirruslabs/cirrus-cli/pull/700/commits/b5590f927f7a17ba0c6fc2586a82aa04a45e80eb.